BUILD

```diff
@@ -68,8 +68,14 @@ deploy_maven(
 
 checkstyle_test(
     name = "checkstyle",
-    targets = [":common"],
-    files = ["BUILD", "deployment.bzl", ".grabl/automation.yml"],
+    include = glob([
```

Comment:
noice

Reply:
how does it handle BUILD targets?

Reply:
eg `:common`

Reply:
Basically it doesn't anymore.

Before this PR: https://github.com/graknlabs/dependencies/pull/199, `checkstyle_test` worked by finding `java_library` targets and scraping their `srcs` property to grab all the Java source files. The decision was taken to do it this way because we erroneously believed `checkstyle` only worked on Java sources. In fact, it can run on any file.

So now, we just skip the 'intermediate' step of going through the `java_library` targets, and go straight for the files themselves.

Reply:
huh! I thought that only one BUILD file could read its set of files, and another BUILD couldn't access them as well or something!